scripts/validate-nimbus-file.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+TARGET_PATH=/tmp/nimbus-cli
+NIMBUS_INSTALL_REMOTE_PATH=https://raw.githubusercontent.com/mozilla/application-services/main/install-nimbus-cli.sh
+NIMBUS_INSTALL_LOCAL_PATH=$TARGET_PATH/install.sh
+
+if [ -d $TARGET_PATH ]; then
+  echo "Directory for \`nimbus-cli\` already exists: $TARGET_PATH"
+else
+  echo "Create directory for \`nimbus-cli\`: $TARGET_PATH"
+  mkdir -p $TARGET_PATH
+
+  echo "Download \`nimbus-cli\` installer from $NIMBUS_INSTALL_REMOTE_PATH to $TARGET_PATH"
+  curl --proto '=https' --tlsv1.2 -sSf -o "$NIMBUS_INSTALL_LOCAL_PATH" "$NIMBUS_INSTALL_REMOTE_PATH"
+
+  echo "Install \`nimbus-cli\`"
```

Comment:
I would maybe check if we've already downloaded nimbus-cli and skip the download/install step in that case but 🤷‍♀️

Reply:
Yeah, only checking for the existence of the `TARGET_PATH` directory is not the safest bet. I’ve added conditions to check if `nimbus-cli` is downloaded and installed in [e55459c](https://github.com/mozilla/blurts-server/pull/4861/commits/e55459c7cd2a5098f6f456eacb16d89fd9e66b5f).